setup.py

```diff
@@ -40,6 +40,7 @@
           "platformdirs>=2.2.0",
           "dataclasses>=0.7;python_version<='3.6'",
           "typing_extensions>=4.0; python_version<'3.11'",
+          "graphviz",
```

Comment:
Hard dependency on graphviz: not happening. Why not move/use the existing functionality for visualizing dot output, e.g. in pymbolic/pytato?

Reply:
What do you think of [cab9e95](https://github.com/inducer/pytools/pull/151/commits/cab9e952cae5b10254679aa79097c9d6a7ca010d)? This uses the dot visualization from pymbolic.

Reply:
Could you prepare a PR to pymbolic that lets it make use of this?

Reply:
Done in https://github.com/inducer/pymbolic/pull/115